src/plugins.py

```diff
@@ -35,7 +35,9 @@ def loadPlugin(self, i):
 		else:
 			return False
 		try: disabled = plugin.DISABLED
-		except: plugin.DISABLED = False
```

Comment:
Ugh... looking at this code makes me barf. I don't know what toxins I was on writing this. This could use a bit of a re-write.